src/java/davmail/exchange/ExchangeSession.java

```diff
@@ -2146,13 +2146,17 @@ protected void fixICS(byte[] icsContent, boolean fromServer) throws IOException
             if (LOGGER.isDebugEnabled() && fromServer) {
                 dumpIndex++;
                 String icsBody = new String(icsContent, StandardCharsets.UTF_8);
+                ICSCalendarValidator.ValidationResult vr = ICSCalendarValidator.validateWithDetails(icsBody); //ifrh

```

Comment:
I would remove the ` //ifrh` from the end of this line and the others you've modified in this file.

Reply:
done